content/_data/changelogs/weekly.yml

```diff
@@ -17712,6 +17712,58 @@
           - url: /security/advisory/2022-09-21/
             title: security advisory
 
+  - version: '2.371'
+    date: 2022-09-26
+    changes:
+      - type: TODO
+        category: developer
+        pull: 7162
+        authors:
+          - dependabot[bot]
+        pr_title: Bump checkstyle from 10.3.3 to 10.3.4
+        message: |-
+          (No proposed changelog)
```

Comment:
```suggestion
      - type: rfe
        category: developer
        pull: 7162
        authors:
          - dependabot[bot]
        pr_title: Bump checkstyle from 10.3.3 to 10.3.4
        message: |-
          Upgrade checkstyle from 10.3.3 to 10.3.4.
```
Should be at bottom of changelog entry since it is developer category, message should not be (no proposed changelog)